app/coffeescripts/views/assignments/VeriCiteSettingsDialog.coffee

```diff
@@ -0,0 +1,68 @@
+define [
```

Comment:
I deleted this file and made TurnitinSettingsDialog work with multiple plagiarism types